bundles/CoreBundle/Resources/translations/de.extended.json

```diff
@@ -721,5 +721,11 @@
     "1x1_pixel_placeholder": "1x1 Pixel Platzhalter",
     "1x1_pixel_placeholder_description": "Gibt nur ein 1x1 Pixel base64-codiertes GIF zurück, falls Sie für eine bestimmte Bedingung kein Bild anzeigen möchten",
     "you_can_drag_the_tabs_to_reorder_the_media_queries": "Sie können die Registerkarten ziehen, um die Priorität der Medienabfragen neu zu ordnen",
-    "classificationstore_group_limitation": "Max. Gruppenelemente"
+    "classificationstore_group_limitation": "Max. Gruppenelemente",
+    "web2print_headlesschrome_puppeteer_documentation": "Puppeteer",
+    "web2print_headlesschrome_documentation": "Dokumentation für Optionen",
```

Comment:
Please remove the German translations as they are managed here: https://poeditor.com/join/project/VWmZyvFVMH